draft-ietf-httpbis-rand-access-live.xml

```diff
@@ -210,24 +232,26 @@
    byte-range-spec = first-byte-pos "-" [ last-byte-pos ]
 </artwork></figure>
       <t>
-      which allows a Client to send a HEAD request with a first-byte-pos and leave last-byte-pos absent. A Server that receives a satisfiable byte-range request (with first-byte-pos smaller than the current representation length) must respond with a 206 status code (Partial Content) with a Content-Range header indicating the currently satisfiable byte range. For example, a Client-issued HEAD request performed against a continuously aggregating representation hosted on a Server could contain a byte-range header of the form:
+      which allows a client to send a HEAD request with a first-byte-pos and leave last-byte-pos absent. A server that receives a satisfiable byte-range request (with first-byte-pos smaller than the current representation length) must respond with a 206 status code (Partial Content) with a Content-Range header indicating the currently satisfiable byte range. For example, a client-issued HEAD request performed against a continuously aggregating representation hosted on a server could contain a byte-range header of the form:
```

Comment:
The "must" here is incorrect.  It can provide the entire representation (for instance, if it didn't support range requests).  Maybe "might".